packages/editor/src/client/blocks/richTextBlock.tsx

```diff
@@ -320,6 +334,17 @@ export const RichTextBlock = memo(function RichTextBlock({
 
         <LinkFormatButton />
         <RemoveLinkFormatButton />
+
+        <ToolbarDivider />
+
+        {'😄 😁 🤯 🎸 ¿ å etc'.split(' ').map((txt, i) => (
```

Comment:
For Emojis: https://github.com/missive/emoji-mart

It looks like there is no "special characters" picker. I suggest we make this into an emoji only picker.

Reply:
Ok, will integrate emoji picker and make an emoji-only picker.